examples/tutorial_dbt_dagster_v2/tutorial_dbt_dagster_v2/assets/dbt_add_partitions.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from dagster import DailyPartitionsDefinition, OpExecutionContext
+from dagster_dbt.asset_decorators import dbt_multi_asset
+from dagster_dbt.cli.resources_v2 import DbtClientV2, DbtManifest
+
+from tutorial_dbt_dagster_v2.assets import raw_manifest
+
+DBT_SELECT_SEED = "resource_type:seed"
+
+manifest = DbtManifest(raw_manifest=raw_manifest)
+
+
+## Case 5: Partition different sections of my dbt graph. This replaces:
+##   - partitions_def
+##   - partition_key_to_vars_fn
+@dbt_multi_asset(manifest=manifest, select=DBT_SELECT_SEED)
+def dbt_seed_assets(context: OpExecutionContext, dbt: DbtClientV2):
+    for event in dbt.cli(["seed"], context=context, manifest=manifest).stream():
```

Comment:
this applies to all of these examples, but

```
yield from dbt.cli(["seed"], context=context, manifest=manifest).steam_dagster_events()
```

feels a lot smoother. we can keep around `stream()` for users that want even more flexibility, but reduce boilerplate for the average user. Potentially we just call `stream_dagster_events` "`stream`", then rename the existing behavior to `stream_raw_events()` or something of the sort (just to spare some keystrokes)

Reply:
This has been added in https://github.com/dagster-io/dagster/pull/14289! Thanks for the suggestion, this improvement saves a line in the compute body implementation in the simple case 🙌